test/lit/binary/strings-nogc.test

```diff
@@ -5,8 +5,5 @@
 (module
  (func $0
   (local $0 stringref)
-  (local.set $0
-   (ref.null none)
```

Comment:
Maybe replace this with a `ref.null string`?

Reply:
Won't work since a `ref.null string` is represented as a `ref.null none`. So you can't represent a null string without GC being enabled. That's a little weird, but I don't think it's actually a problem.

Reply:
But then how can you assign a null to a stringref..?

Reply:
By enabling gc ;)

Reply:
Or by assigning from a get of an uninitialized stringref local if you really really want to do it, but truly the real answer is to enable gc.

Reply:
:smile: 

But to make sure I follow, that is a spec issue, not a Binaryen issue?

Anyhow, I guess it can be done even without gc...
```wat
(func $ref.null_string (result stringref)
  (local $x stringref)
  (local.get $x)
)
```
And just write `(call $ref.null_string)` instead of `(ref.null string)`...

Reply:
(ah, we raced on the last comments :rofl: )

Reply:
Right, the stringref spec implicitly depends on WasmGC, so we're just reflecting that here.